tools/release-executor.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+# Similar to the build-executor script, except this one will also push the
+# image to the container registry.
+
+if [ ! -f "EXECUTOR_VERSION" ]; then
+  echo "Could not find EXECUTOR_VERSION file" >& 2
+  exit 1
+fi
+
+VERSION=`cat EXECUTOR_VERSION`
+
+echo "Building and tagging prairielearn/executor:$VERSION"
+
+docker build ./images/executor --tag "prairielearn/executor:$VERSION"
+
+if [ $? -ne 0 ]; then
+  echo "Image failed to build; not pushing to container registry." >& 2
+  exit 2
+fi
+
+echo "Pushing prairielearn/executor:$VERSION to image repository"
+
+docker push "prairielearn/executor:$VERSION"
```

Comment:
I think we are going to want to use ECR to store these images rather than hub.docker. We now have this variable: https://github.com/PrairieLearn/PrairieLearn/blob/6b15c9dea5232bd37a7fcb9f1cb0e5f591b9291a/lib/config.js#L94 which is set to something like `123456890.dkr.ecr.us-east-2.amazonaws.com` in production. We'll want to push executor images to a tag like `123456890.dkr.ecr.us-east-2.amazonaws.com/prairielearn/prairielearn` and also pull from something like this.

Reply:
Just to clarify, the only thing we need to change in the code is the `docker pull` which should stick `${config.cacheImageRegistry}/` onto the front of the image name. In our deploy scripts we'll need to also push to this registry. Maybe the easiest way to handle this is to have `release-executor.sh` take a commandline arg which is the registry name (if empty, just ignore it)?

Reply:
`lib/code-caller-docker` will now pull from `config.cacheImageRegistry` if it's configured, and the new `tools/executor/release.js` script will dual-publish to both the Docker registry and ECR.